ajax.sms.php

```diff
@@ -2,16 +2,32 @@
 
 use UKMNorge\Arrangement\Arrangement;
 use UKMNorge\Database\SQL\Query;
+use UKMNorge\Wordpress\Nyhet;
+use UKMNorge\DesignWordpress\Environment\Posts;
+use UKMNorge\DesignWordpress\Environment\Wordpress;
+
 
 require_once('UKM/Autoloader.php');
 
 function UKMSMS_ajax(){
 	switch($_POST['SMSaction']){
-		case 'log':				UKMSMS_ajax_log();
+		case 'log':
+			UKMSMS_ajax_log();
+			break;
+		case 'nyhetssak':
+			getNyhetssaker();
+			break;
 	}
 	die();
 }
 
+function getNyhetssaker() {
+	$posts = new Posts();
+	Wordpress::setPosts($posts);
```

Comment:
Hva gjør denne @slogan-15 ? 😊

Reply:
Den ble brukt her for å hente nyheter på frontsiden, så kopierte jeg det @mariusmandal 😊 

https://github.com/UKMNorge/UKMDesignWordpress/blob/3a139f086303e340f01a25afc41f2e172b4d18ed/Controller/Arrangement/front-page.php#L10